activerecord/lib/active_record/connection_adapters.rb

```diff
@@ -25,7 +25,7 @@ module ConnectionAdapters
       autoload :ReferenceDefinition
     end
 
-    autoload_at "active_record/connection_adapters/abstract/connection_pool" do
+    autoload_at "active_record/connection_adapters/abstract/connection_handler" do
```

Comment:
This can also be moved to `autoload_under`

Reply:
Oops, you're right. Fixed!